resources/newssite/news-next/src/workload-test.mjs

```diff
@@ -0,0 +1,41 @@
+import { BenchmarkStep, BenchmarkSuite, forceLayout, getElement } from "speedometer-utils/workload-testing-utils.mjs";
```

Comment:
Don't we need to import from benchmark.mjs and helpers.mjs now that they're split?

Reply:
the imported files re-exports them

Reply:
Do we really need that? Directly importing from benchmark.mjs & helpers.mjs seem better?

Reply:
changed to two imports